test/e2e/tests/editor/python-scratch-file-execution.test.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2025 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { test, tags } from '../_test.setup';
+
+test.use({
+	suiteId: __filename
+});
+
+test.describe('Python Scratch File', { tag: [tags.WEB, tags.EDITOR, tags.WIN] }, () => {
+	test('Verify that lines in a python scratch file with magics can be executed', async function ({ app, python, runCommand }) {
+
+		const filename = 'Untitled-1';
+		await test.step('Create a new python scratch file with code and a magic', async () => {
+			await runCommand('python.createNewFile');
+			await app.workbench.editor.waitForTypeInEditor(filename, 'import pandas as pd');
+			await app.code.driver.page.keyboard.press('Enter');
+			await app.code.driver.page.keyboard.press('Enter');
+			await app.workbench.editor.waitForTypeInEditor(filename, 'pip install pyarrow');
+		});
+
+		await test.step('Exexcute first line of code', async () => {
+			await app.workbench.editor.clickOnTerm(filename, 'import', 1, true);
+			await app.code.driver.page.keyboard.press('ArrowLeft');
+
+			await app.code.driver.page.keyboard.press('Control+Enter');
+		});
+
+		// Ensure "deprecated" does not appear in the console
+		await app.workbench.console.waitForConsoleContents('deprecated', { timeout: 5000, expectedCount: 0 });
```

Comment:
Alternatively we could add some `print()` statement instead of `import pandas` and that should appear it should appear in the console.

Reply:
Ok.  Went ahead and changed to:
print("test")
Left the check to make sure "deprecated" doesn't appear for good measure.